### PR TITLE
Tip to configure antivirus on Windows

### PR DESCRIPTION
**Pull request by filippobottega**
What is the purpose of this change? What does it change?
--------------------------------------------------------
Added a tip to configure antivirus to exclude restic process.
Just a little improvement of the documentation.

Was the change discussed in an issue or in the forum before?
------------------------------------------------------------

Checklist
---------

- [ ] I have read the [Contribution Guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches)
- [ ] I have added tests for all changes in this PR
- [ ] I have added documentation for the changes (in the manual)
- [ ] There's a new file in `changelog/unreleased/` that describes the changes for our users (template [here](https://github.com/restic/restic/blob/master/changelog/TEMPLATE))
- [ ] I have run `gofmt` on the code in all commits
- [ ] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits)
- [X] I'm done, this Pull Request is ready for review

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -172,4 +172,4 @@ The following may work:
 Why does restic perform so poorly on Windows?
 ---------------------------------------------
 
-In some cases the real-time protection of antivirus software can interfere with restic's operations. If you are experiencing bad performance you can try to temporarily disable your antivirus software to find out if it is the cause for your performance problems.
+In some cases the real-time protection of antivirus software can interfere with restic's operations. If you are experiencing bad performance you can try to temporarily disable your antivirus software to find out if it is the cause for your performance problems. If you find out that the antivirus is the cause, and you want to gain maximum performance, you have to add restic process (the full path) to the exclusions list.
```
Comment (moritzdietz) [inline doc/faq.rst]:
```suggestion
In some cases the real-time protection of antivirus software can interfere with restic's operations. If you are experiencing bad performance you can try to temporarily disable your antivirus software to find out if it is the cause for your performance problems. If you are certain that the antivirus software is the cause for this and you want to gain maximum performance, you have to add the restic binary to an exclusions list within the antivirus software.
```

**Review by moritzdietz — CHANGES_REQUESTED** [review]
Hey! Thanks for taking the time to update the docs.
I have added some suggestions to fix some grammar issues as well as keep the general style of writing with the previous sentence.
I also removed the bracketed part as antivirus programs sometimes require the full path and sometimes just the name etc. so we just can't know if it is the full path. By saying that one has to add the restic binary to an exclusions list, there is no issue with the wording.

Other than that, LGTM.

**Comment by hickford** [comment]
Windows Defender is the default and most common antivirus software on Windows. This worked for me:

image

**Comment by ProactiveServices** [comment]
@hickford I'd suggest that it would be more useful to diagnose and resolve why Windows Defender is causing problems with restic. I haven't come across this myself, with that combination, on several machines. There's a mention of it [here](https://forum.restic.net/t/windows-defender-causes-10x-slowdown/925/13) but that's over a year old.

**Comment by moritzdietz** [comment]
Honestly, I don't think restic can do anything about this.
If you google windows defender problems in regard to slowdowns this is a topic that comes up with lots of other applications.

**Review by rawtaz — CHANGES_REQUESTED** [review]
I don't think this addition will do any harm, so let's merge it. But before we do that, can you please squash all the commits into one? Thanks!

**Comment by filippobottega** [comment]
Sorry, I don't understand if I have to do something or not...
What does it means "squash all the commits into one"? How can I do it?

**Comment by fd0** [comment]
I've squashed the commits and merged this as 6c700f95b5dd54a5fac47fc12253828462a5f8b2, thanks for your contribution!

**Pull request closed without merging**